src/algo/LinkedList.js

```diff
@@ -294,13 +299,20 @@ export default class LinkedList extends Algorithm {
 	reset() {
 		this.size = 0;
 		this.nextIndex = this.initialIndex;
+		this.infoLabelID = this.nextIndex++;
```

Comment:
```suggestion
```

Reply:
Same idea as the other ones, except resetIndex is called this.initialIndex here haha

Reply:
Thanks for explaining this, now I know what resetIndex actually does!